taichi/backends/cpu/jit_cpu.cpp

```diff
@@ -129,15 +129,23 @@ class JITSessionCPU : public JITSession {
 
   void *lookup(const std::string Name) override {
     std::lock_guard<std::mutex> _(mut);
+#ifdef __APPLE__
+    auto symbol = ES.lookup(all_libs, Mangle(Name));
```

Comment:
Just a curious question, I'm not familiar with what type of mangling LLVM will do on mac for IR, it's it just some sort of `_` or `__`?

Reply:
My experiment shows the mangling results of function `foo` will be `_foo{i}`(i will start from 0)